ipaclient/install/ipa_client_automount.py

```diff
@@ -340,14 +340,17 @@ def configure_nfs(fstore, statestore, options):
 
 
 def configure_automount():
+    statestore = sysrestore.StateFile(paths.IPA_CLIENT_SYSRESTORE)
     try:
         check_client_configuration()
     except ScriptError as e:
-        print(e.msg)
-        sys.exit(e.rval)
+        # Are we being called from ipa-client-install?
+        automount = statestore.get_state('installation', 'automount')
```

Comment:
I like your approach with the additional state variable. The flow is a bit hard to follow, though. How about you move the check for the new state var one level up and skip the client configure script when the state is set?

```suggestion
    if not statestore.get_state('installation', 'automount'):
        # not called from ipa-client-install
        try:
            check_client_configuration()
        except ScriptError as e:
            print(e.msg)
            sys.exit(e.rval)
```

Reply:
Yes, that is much simpler. Done.